### DISCUSSION TITLE
New Cover Art and scheme arts

### DISCUSSION THREAD

**Discussion by nimsandu** [nimsandu/spicetify-bloom#275] [Show and tell]
Feel free to provide your thoughts on this designs. cover art is completed as follows. other great things are on Their way here!
@SunsetTechuila @windowz414
![prealfa](https://github.com/nimsandu/spicetify-bloom/assets/57834844/2500070a-f925-43c8-8bb3-e528fa1b4454)

**Comment by nimsandu** [comment]
i wanted it to be simple what do you think?

**Comment by SunsetTechuila** [comment]
Very nice! I mean **really** nice, looks cool!

A few comments and suggestions:
- If you include fluent scheme you need to show it is transparent
- I personally don't think that we need a lyrics page on the cover art, better to keep it just in the screenshots section 
- It will be better to use only Segoe UI Variable (I'm pretty sure the header at least has a different font)
- Top logo shape looks weird, maybe remove it completely?
- Background is too simple for me
- Window screenshots are too small

**Comment by SunsetTechuila** [comment]
offtop: can you please provide this button class/id?
![image](https://github.com/nimsandu/spicetify-bloom/assets/115353812/ceba9716-a334-49e9-bf64-05c5e80347a9)
